### ISSUE TITLE
ESXi: Support linked clones

### ISSUE THREAD

**Issue by moshloop** [kubernetes-sigs/cluster-api-provider-vsphere#290]
Depends on #217

**Comment by akutz** [comment]
There are no current plans to support ESXi. While that support was added recently, any net-new work around ESXi is not a priority. I am going to close this issue as this will not be addressed directly in the near future.

/close

**Comment by akutz** [comment]
Hmm. I'm going to reopen this as I'm unclear on the distinction between this and #289 

/reopen
